### DISCUSSION TITLE
How to download requirements behind company proxy

### DISCUSSION THREAD

**Discussion by n4rk0o** [reflex-dev/reflex#3526] [Q&A] [answered]
Hello,
I started to use reflex.dev and with direct internet connection it's downloading NPM requirements but when I'm trying to install it with limited internet I get error messages because of the company restrictions about internet downloads.

Is there a way to configure proxy properly within reflex CLI or do I need to modify some files to be able to download through a proxy?

Thanks.

**Comment by seewindcn** [comment, marked as answer]
there are a lot of ways for this; list some below:
1. maybe you can try to use http_proxy(https_proxy) environ, lot of tools support this way; (I don't have try this for reflex)
2. you can use global proxy tools when http_proxy isn't work; proxy tools like clash, tun2proxy.....
3. if you have some computer's internet connection is ok, you can use "tailscale" make your all computers in your own private network, and config the compony computer use other computer as "exit node", so compony computer's network will through the "exit node", bypass the company limit;

**Reply by n4rk0o** [reply]
Thanks for you suggestions
